README.md

```diff
@@ -53,7 +74,7 @@ player.pause()
 player.quit()
 ```
 
-Playing a file from a URL (courtesy of @jappe999)
+Playing a file from a URL by downloading the network file (courtesy of @jappe999)
```

Comment:
Cool, I'll add a note on the bottom of the README for suggested usage patterns like this one, but I don't think they should clutter the main usage example.